src/platform_impl/linux/window.rs

```diff
@@ -215,7 +215,15 @@ impl Window {
     window.set_visible(attributes.visible);
     window.set_decorated(attributes.decorations);
 
-    window.set_keep_above(attributes.always_on_top);
+    if attributes.always_on_bottom && attributes.always_on_top {
+      log::warn!(
+        "Always on top and always on bottom, are both specified, you should only set one."
+      );
+    } else if attributes.always_on_bottom || attributes.always_on_top {
+      window.set_keep_below(attributes.always_on_bottom);
+      window.set_keep_above(attributes.always_on_top);
+    }
+
```

Comment:
The warning is not needed anymore, since in your latest commit, the builder will disable one of them
```suggestion
    if attributes.always_on_bottom {
      window.set_keep_below(attributes.always_on_bottom);
    }

    if attributes.always_on_top {
      window.set_keep_above(attributes.always_on_top);
    }

```

Reply:
a563d89

I also [set `ALWAYS_ON_BOTTOM` flag](https://github.com/tauri-apps/tao/pull/522/files#diff-edacb93792c7cf2d4cfe6f4ff2259781ed3c13e060fcc381e9cbb30ce481a336R858) when attribute is set on Windows